flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/MFD/A320_Neo_MFD.js

```diff
@@ -162,29 +163,25 @@ class A320_Neo_MFD_MainPage extends NavSystemPage {
     }
 
     onUpdate() {
-        const currentKnobValue = SimVar.GetSimVarValue("LIGHT POTENTIOMETER:" + this.potIndex, "number");
-        if (currentKnobValue <= 0.0) {
-            this.selfTestLastKnobValue = currentKnobValue;
-            return;
-        }
```

Comment:
I removed this `return;` as it caused the chrono to not initialise when pressing the chrono button with the DU off. In the real aircraft the DMC updates the chrono and thus the button should also work with the DU off.